classes/admin-user-profile.php

```diff
@@ -127,7 +127,7 @@ function add_meta_fields( $user ) {
                 <tr>
                     <th><?php esc_html_e( 'Store URL', 'dokan-lite' ); ?></th>
                     <td>
-                        <input type="text" name="dokan_store_url" class="regular-text" id="seller-url" value="<?php echo esc_attr( $user->data->user_nicename ); ?>"><strong id="url-alart-mgs"></strong>
+                        <input type="text" name="dokan_store_url" vendor="<?php echo $user->ID; ?>" class="regular-text" id="seller-url" value="<?php echo esc_attr( $user->data->user_nicename ); ?>"><strong id="url-alart-mgs"></strong>
```

Comment:
You are using a custom attribute for vendor which is non-standard. Use a data property instead.